src/instructlab/config.py

```diff
@@ -19,8 +19,8 @@
 
 DEFAULT_API_KEY = "no_api_key"
 DEFAULT_CONFIG = "config.yaml"
-DEFAULT_MODEL = "merlinite-7b-lab-Q4_K_M"
-DEFAULT_MODEL_PATH = f"models/{DEFAULT_MODEL}.gguf"
+DEFAULT_MODEL = "models/merlinite-7b-lab-Q4_K_M.gguf"
+DEFAULT_MODEL_PATH = "models/merlinite-7b-lab-Q4_K_M.gguf"
```

Comment:
Since both `DEFAULT_MODEL` and `DEFAULT_MODEL_PATH` share the same value. Why keep both?

Reply:
Adding a todo, I think this  consolidation is out of scope for this PR as that would involve changing cmd flags since these vars auto populate those, right?

```# TODO: Consolidate --model and --model-path into one --model-path flag since we always need a path now
```

Reply:
I'm fine with that, another small request, could you please open an issue so we can track it more easily? Thanks!

Reply:
or just this and skip the issue ...

```suggestion
DEFAULT_MODEL_PATH = DEFAULT_MODEL = "models/merlinite-7b-lab-Q4_K_M.gguf"
```